10_deploy_rook.sh

```diff
@@ -43,6 +56,9 @@ sed 's/namespace: rook-ceph/namespace: openshift-storage/' toolbox.yaml > toolbo
 sed -i "s@rook/ceph:master@rook/ceph:$ROOK_VERSION@" toolbox-modified.yaml
 oc create -f toolbox-modified.yaml
 
+oc wait --for condition=ready  pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
+oc -n openshift-storage exec $(oc -n openshift-storage get pod --show-all=false -l "app=rook-ceph-tools" -o jsonpath='{.items[0].metadata.name}') -- ceph mgr module enable pg_autoscaler
```

Comment:
This line gives the below error 

```
+ oc -n openshift-storage exec rook-ceph-tools-5b85446c4b-2fflw -- ceph mgr module enable pg_autoscaler
Error ENOENT: all mgr daemons do not support module 'pg_autoscaler', pass --force to force enablement
command terminated with exit code 2
```

@leseb ^^^

But if after this error we run the command from CLI it doesnt show any error and auto_scaler is enabled on ceph cluster.